azure/cluster.tf

```diff
@@ -0,0 +1,140 @@
+#
+# Redpanda broker VMs
+#
+
+resource "azurerm_linux_virtual_machine" "redpanda" {
+  name                 = "redpanda_broker${count.index}"
+  computer_name        = "redpanda${count.index}"
+  count                = var.vm_instances
+  resource_group_name  = azurerm_resource_group.redpanda.name
+  location             = azurerm_resource_group.redpanda.location
+  # availability_set_id  = azurerm_availability_set.redpanda.id
```

Comment:
Is there a reason this is commented out?

Reply:
Due to general lack of availability it’s difficult to acquire Lsv2 instances in the same availability set, which causes Terraform to constantly receive AllocationFailed error messages. Commenting this out widens the hardware Azure can select from (meaning that Lsv2 instances can be deployed on the same server). I'll put this back in for now as ideally we want the brokers to be in an availability set to benefits from the 99.95% SLA.